Update mixs-templates/README.md (#786)
GenomicsStandardsConsortium/mixs: mixs-templates/README.md

## MIxS (meta)data collection spreadsheet templates

> In this folder, the GSC provides Microsoft Excel (.xlsx) spreadsheets for collecting metadata that is roughly aligned with the MIxS standard. THe GSC appreciates that spreadsheets like these can be very useful for preparing data to be submitted to an INSDC database. Users should be aware of the limitations of these spreadsheets, listed below.

Reviewer: Questions:
* what does "roughly aligned" mean?
* typo "THe" --> "The"

Author: thanks @sujaypatil96 I really appreciate this review and apologize for not replying sooner

I think (!) "roughly aligned" means that the templates provide prompts (in the form of column headers) to use the terms associated with the template's class, but it doesn't do anything to ensure that valid values are provided. I'll fix that up.

> In this folder, the GSC provides Microsoft Excel (.xlsx) spreadsheets that can be used as templates for collecting metadata according to the MIxS standard. The GSC appreciates that spreadsheets like these can be very useful for preparing data to be submitted to an INSDC database. Users should be aware of the limitations of these spreadsheets, listed below.

Reviewer 2: ```suggestion
In this folder, the GSC provides Microsoft Excel (.xlsx) spreadsheets that can be used as templates for collecting metadata according to the MIxS standard. 

The GSC appreciates that spreadsheets like these can be very useful for use in the field and lab, as well as for preparing data to be submitted to an INSDC database. However, users should be aware of the limitations of these spreadsheets, listed below.
```

> Each of the templates in this folder are blank spreadsheet templates with only the header row populated with the names of the terms that are associated with a particular [checklist](https://sujaypatil96.github.io/mixs/#checklists), [extension](https://sujaypatil96.github.io/mixs/#extensions), or [combination](https://sujaypatil96.github.io/mixs/combinations/) (of one checklist plus one extension). 

Reviewer: Here we have started using the word "templates" without clarifying what "templates" are exactly? I think we should use either "spreadsheet files" or "templates" uniformly throughout the text?

Author: The heading of the document is

## MIxS (meta)data collection spreadsheet templates

and the file we're talking about is `mixs-templates/README.md`.

Template appears in the document 8 times and spreadsheet appears 9 times

But you are right, I should use consistent and defined terminology! I edited the first paragraph so that "template" and "spreadsheet" are introduced at the same time.

> Each of the files in this folder and its subfolders are blank template spreadsheets, with only the header row populated with the names of terms associated with a particular [Checklist](https://sujaypatil96.github.io/mixs/#Checklists), [Extension](https://sujaypatil96.github.io/mixs/#Extensions), or [Combination](https://sujaypatil96.github.io/mixs/combinations/) (of one Checklist plus one Extension). 

Reviewer 2: The links here point to @sujaypatil96 's page - this should point to a GSC official site/repo

Author: thanks

* There is one template for each *checklist*, *extension* and *combination* in the [schema](src/mixs/schema/mixs.yaml).

* The organization of files in this folder is such that there is one checklist subfolder per checklist, and an extensions subfolder.
  * The checklist subfolders contain the template for that checklist, as well as the templates for the combinations that have been derived from that checklist. 
  * The extensions subfolder contains the templates for all the extensions in the schema.

### Limitations:

> - The use of Excel spreadsheets is a violation of the 5-star linked data standards, which require the use of an open format like TSV or CSV

Reviewer: Provide a link to the 5-star linked data standards?

Found this after googling: https://www.w3.org/2011/gld/wiki/5_Star_Linked_Data

> - Excel files do not play well with GitHub: it is very difficult to view them or diff them within GH, and they take up more disk space than TSV or CSV

Reviewer: GH --> GitHub

> - The use of Excel spreadsheets is a violation of the [5-star linked data standards](https://www.w3.org/2011/gld/wiki/5_Star_Linked_Data), which require the use of an open format like TSV or CSV

Reviewer 2: ```suggestion
- The use of Excel spreadsheets is convenient, but is low on the [5-star linked data framework](https://www.w3.org/2011/gld/wiki/5_Star_Linked_Data), and not conducive to good implementations of the FAIR Principles. Open formats like TSV or CSV are preferred, and graph-friendly formats like JSON-LD or RDF encouraged.
```

> - These spreadsheets confuse the **specification of a standard** and an **application that uses the standard** to collect data. THe spreadsheets do not apply any validation to the entered metadata. They do provide one data entry convenience: pulldown menus are provided for filling out columns when there is a brief [enumeration](https://sujaypatil96.github.io/mixs/enumerations/) of permissible values, like [window_type](https://sujaypatil96.github.io/mixs/0000856/), but this [feature is disabled](https://linkml.io/linkml/generators/excel.html) if the total character length of the permissible values is more than 255, like [fao_class](https://sujaypatil96.github.io/mixs/0001083/).

Reviewer: * typo "THe" --> "The"
* also probably not the best idea to provide links using my fork of mixs? 
  * https://sujaypatil96.github.io/mixs/enumerations/ --> https://genomicsstandardsconsortium.github.io/mixs/enumerations/
  * https://sujaypatil96.github.io/mixs/0000856/ --> https://genomicsstandardsconsortium.github.io/mixs/0000856/
  * https://sujaypatil96.github.io/mixs/0001083/ --> https://genomicsstandardsconsortium.github.io/mixs/0001083/

Author: Great review, @sujaypatil96 . Apologies for leaving so much junk in there.

> - These spreadsheets confuse the **specification of a standard** and an **application that uses the standard** to collect data. The spreadsheets do not apply any validation to the entered metadata. They do provide one data entry convenience: pulldown menus are provided for filling out columns when there is a brief [enumeration](https://genomicsstandardsconsortium.github.io/mixs/enumerations/) of permissible values, like [window_type](https://genomicsstandardsconsortium.github.io/mixs/0000856/), but this [feature is disabled](https://linkml.io/linkml/generators/excel.html) if the total character length of the permissible values is more than 255, like [fao_class](https://genomicsstandardsconsortium.github.io/mixs/0001083/).

Reviewer 2: ```suggestion
- These spreadsheets confound the **specification of a standard** and an **application that uses the standard** to collect data. The spreadsheets do not apply any validation to the entered metadata. They do provide one data entry convenience: pulldown menus are provided for filling out columns when there is a brief [enumeration](https://genomicsstandardsconsortium.github.io/mixs/enumerations/) of permissible values, like [window_type](https://genomicsstandardsconsortium.github.io/mixs/0000856/), but this [feature is disabled](https://linkml.io/linkml/generators/excel.html) if the total character length of the permissible values is more than 255, like [fao_class](https://genomicsstandardsconsortium.github.io/mixs/0001083/).
```
